File: Code Chef/C Programming/600-699/613 Bus Seat Numbering SEATNUMBER.c
//n --> seat no.

//if n is in range 1-10 Lower Double
//if n is in range 11-15 Lower Single
//if n is in range 16-25 Upper Double
//if n is in range 26-30 Upper Single
#include <stdio.h>
int main(void)
{
	int t,n;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d",&n);
	    if(n<16)
	    {
	        if(n<11)
	        {
	            printf("Lower Double\n");
	        }
	        else
	        {
	            printf("Lower Single\n");
	        }
	    }
	    else
	    {
	        if(n<26)
	        {
	            printf("Upper Double\n");
	        }
	        else
	        {
	            printf("Upper Single\n");
	        }
	        
	    }
	}
	return 0;
}

File: Code Chef/JavaScript/600-699/613 Bus Seat Numbering SEATNUMBER.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => Number(strItem.trim()));
    
    return intArray;
};

const findSeatType = (seatNum) => {
    (seatNum <= 10)
        ? console.log('Lower Double')
        : (seatNum <= 15)
            ? console.log('Lower Single')
            : (seatNum <= 25)
                ? console.log('Upper Double')
                : console.log('Upper Single');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((seatNum) => {
        findSeatType(seatNum);
    });
});